standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -888,35 +887,13 @@ public void cleanEmptyAbortedAndCommittedTxns() throws MetaException {
           */
         long lowWaterMark = getOpenTxnTimeoutLowBoundaryTxnId(dbConn);
 
-        String s = "SELECT \"TXN_ID\" FROM \"TXNS\" WHERE " +
+        String s = "DELETE FROM \"TXNS\" WHERE " +
```

Comment:
Could we move sql to constants (see `TxnQueries`) and use prepared statement?

Reply:
hi @deniskuzZ, the query is not hardcoded fully.
we have some txn state fields in middile of the query.
TxnStatus.ABORTED,  and TxnStatus.COMMITTED. So better to keep like this for better readable purpose.

Reply:
what do you mean? they only var here is `lowWaterMark` that should be passed to prepared statement

Reply:
done